### ISSUE TITLE
Numerical font-weight not parsed correctly in font shorthand

### ISSUE THREAD

**Issue by mganss** [AngleSharp/AngleSharp.Css#56]
## Description

A `font` shorthand declaration is not parsed correctly if it contains a numerical `font-weight`.

## Steps to Reproduce

```c#
var p = new HtmlParser(new HtmlParserOptions(), BrowsingContext.New(new Configuration().WithCss(new CssParserOptions())));
var dom = p.ParseDocument(@"<html><body><div style=""font: 400 12px serif"">Test</div></body></html>");
var div = dom.QuerySelector("div");
var style = div.GetStyle(); // -> style.Length == 0
var css = style.CssText; // -> ""
```

It works if `font-weight` is specified using a constant, e.g. `font: bold 12px serif`.

## Possible Solution

It seems the numerical case is not considered here:
https://github.com/AngleSharp/AngleSharp.Css/blob/e59a6e7d9afd2ee716f0a5a7c0535cfc0e9001c7/src/AngleSharp.Css/Declarations/FontDeclaration.cs#L61-L65

**Comment by FlorianRappl** [comment]
Yes I guess that's right. Can you supply a PR?

Thanks - much appreciated! :beers:

**Comment by FlorianRappl** [comment]
Available in 0.14.2!
